incubator/uchiwa/templates/NOTES.txt

```diff
@@ -0,0 +1,5 @@
+Getting Started:
+
+1. Uchiwa can be accessed at the following address from within your cluster: http://{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local:{{ .Values.httpPort }}
```

Comment:
Since most users will not be coming from inside the cluster, can you show how you would access it from outside the cluster?

The Jenkins chart has a good example.